pkg/process/util/network_filter.go

```diff
@@ -0,0 +1,74 @@
+package util
+
+import (
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/stretchr/stew/slice"
+)
+
+// Connection holds a user-defined blacklisted IP, CIDR, and ports
+type Connection struct {
+	IP    string
+	CIDR  *net.IPNet
```

Comment:
Can we model single IPs as CIDRs with prefix lengths of 32? It looks like you'll never construct a filter with both IP and CIDR set.